src/models/accountrole.js

```diff
@@ -0,0 +1,24 @@
+const { Model } = require("sequelize");
```

Comment:
File names are not renamed, as I asked before. Same applies for other files with models

Reply:
I do not understand why, because I renamed these.
<img width="226" alt="image" src="https://user-images.githubusercontent.com/63366123/159699660-9efd19f2-8587-4dfa-9c94-f01c213ec576.png">

Reply:
Please rename them to camelCase, not Pascal one

Reply:
yes